R/R/prophet.R

```diff
@@ -1160,4 +1160,52 @@ plot_yearly <- function(m, uncertainty = TRUE, yearly_start = 0) {
   return(gg.yearly)
 }
 
+#' Return a List having posterior predictive samples
+#'
+#' @param m Prophet model object.
+#' @param future Dataframe that extends forward from the end of m$history for the
+#' requested number of periods.
+#' @return list of sim.values : posterior predictive samples,
+#'                 yhat.mean : mean of predictive samples,
+#'                 lower.value : lower bound of uncertainty interval 
+#'                 upper.value : upper bound of uncertainty interval                
+#' @export
+posterior_analysis <- function(future, m) {
```

Comment:
Let's name this `predictive_samples` and have the input variable names match those of `predict.prophet`. In this commit this function is also put at the end of the file near the plotting functionality; let's move it up to be near all of the other prediction related functions. Right after `predict_uncertainty` would be great.